client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features.tsx

```diff
@@ -50,4 +50,28 @@ const EmailProviderStackedFeatures: FunctionComponent< EmailProviderStackedFeatu
 	);
 };
 
-export default EmailProviderStackedFeatures;
+export interface EmailProviderStackedFeaturesProps {
+	features: TranslateResult[];
+}
+
+export const EmailProviderStackedFeaturesToggleButton: FunctionComponent< EmailProviderStackedFeaturesToggleButton > = (
+	props
+) => {
+	const translate = useTranslate();
+	const { handleClick, isRelatedContentExpanded } = props;
+
+	return (
+		<button className="email-provider-stacked-features__toggle-button" onClick={ handleClick }>
```

Comment:
Question: is there a reason you're using a plain `button` instead of the general-purpose `Button` component?

Reply:
Because it Button adds some styling that I don't want, basically the 3d styling, I've realised that button does not add any styling, and that's why before was a span, because Button added 3d styling 🙂

Reply:
TBH, I would rather use `Button` and override the specific styles you don't want. That way we get more default and consistent behaviour for buttons in general. I also think that most of what you want is covered by the `borderless` attribute -- it's not clear to me what 3d styling you're referring to.